### ISSUE TITLE
Doesn't work under iOS 10

### ISSUE THREAD

**Issue by excitement-engineer** [granoff/Lockbox#56]
When running version 3.0.2 under iOS 10 I get the error: 

```
[Lockbox setData:forKey:accessibility:] [Line 167] SecItemAdd failed for key com.xxx: -34018
```

I also tried running the same code under iOS 9.3.1 and it seems to work fine.

**Comment by granoff** [comment]
Did you enable *Keychain Sharing* in your project? That is required now under iOS10. And this is in the README as well.

**Comment by granoff** [comment]
Fantastic! 

I looked to see if the Podspec had a way to indicate that certain capabilities were required, like you can do for libraries and frameworks, but it does not appear so at the moment. That would prevent this issue in the first place! :-)

**Comment by excitement-engineer** [comment]
That would be really useful indeed! Hopefully, this functionality will be added in the Podspec in the future!

However, the README does state it clearly though so hopefully it is not an issue to other people.
